checker/jtreg/stubs/general/Driver.out

```diff
@@ -0,0 +1,4 @@
+- compiler.warn.proc.messager: StubParser: No supported annotations found! This likely means stub file /Users/smillst/jsr308/checker-framework/checker/jtreg/stubs/general/MyStub.astub doesn't import them correctly.
```

Comment:
Before this change, 
`warning: StubParser: Unknown annotation: @SuppressWarnings("something")`
would have also been issued.